packages/blade/src/components/Input/OTPInput/OTPInput.tsx

```diff
@@ -265,7 +265,7 @@ const OTPInput = ({
           flex={1}
           marginLeft={index == 0 ? 'spacing.0' : 'spacing.3'}
           key={`${inputId}-${index}`}
-          maxWidth={platform === 'onDesktop' ? 36 : 40} // TODO: use size tokens
+          maxWidth={platform === 'onDesktop' ? makeSize(36) : makeSize(40)} // TODO: use size tokens
```

Comment:
```suggestion
          maxWidth={platform === 'onDesktop' ? '36px' : '40px'} // TODO: use size tokens
```

??

Reply:
this also reminds me  - is `makeSize` function inside utils relevant now as we accept string values?

Reply:
We always accepted string values. I assumed `makeSize` utility was for usecase where we decide to change the default units for our sizing and spacing tokens in future so that will still apply 🤔